pkg/sql/logictest/testdata/logic_test/privilege_builtins

```diff
@@ -834,43 +834,46 @@ NULL
 
 ## has_table_privilege
 
-query BBBBBBB
+query BBBBBBBB
 SELECT has_table_privilege(12345, 'SELECT'),
        has_table_privilege(12345, 'INSERT'),
        has_table_privilege(12345, 'UPDATE'),
        has_table_privilege(12345, 'DELETE'),
        has_table_privilege(12345, 'TRUNCATE'),
        has_table_privilege(12345, 'REFERENCES'),
-       has_table_privilege(12345, 'TRIGGER')
+       has_table_privilege(12345, 'TRIGGER'),
+       has_table_privilege(12345, 'RULE')
 ----
-NULL  NULL  NULL  NULL  NULL  NULL  NULL
+NULL  NULL  NULL  NULL  NULL  NULL  NULL  NULL
 
-query BBBBBBB
+query BBBBBBBB
 SELECT has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'SELECT'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'INSERT'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'UPDATE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'DELETE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'TRUNCATE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'REFERENCES'),
-       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'TRIGGER')
+       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'TRIGGER'),
+       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 'pg_type'), 'RULE')
 ----
-true  false  false  false  false  true  false
+true  false  false  false  false  true  false  false
 
-query BBBBBBB
+query BBBBBBBB
 SELECT has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'SELECT'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'INSERT'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'UPDATE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'DELETE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'TRUNCATE'),
        has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'REFERENCES'),
-       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'TRIGGER')
+       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'TRIGGER'),
+       has_table_privilege((SELECT oid FROM pg_class WHERE relname = 't'), 'RULE')
 ----
-true  true  true  true  true  true  true
+true  true  true  true  true  true  true  true
```

Comment:
hm, why does this end up as `true` for RULE? it doesn't seem like RULE was granted anywhere

Reply:
My mistake,  will update it.